src/bregman.jl

```diff
@@ -124,15 +127,29 @@ function bregman(funobj::Function, x::AbstractArray{T}, options::BregmanParams,
         # Update z variable
         @. z = z + d
         # Get λ at first iteration
-        i == 1 && (λ = abs(T(quantile(abs.(z), options.quantile))))
+        if i == 1
```

Comment:
This is very ugly, it takes more space than the actual algorithm, make a proper function with dispatch for this.